flight/Modules/Stabilization/stabilization.c

```diff
@@ -520,8 +520,8 @@ static void stabilizationTask(void* parameters)
 					rateDesiredAxis[i] = bound_sym(raw_input * settings.ManualRate[i], settings.ManualRate[i]);
 
 					// Zero integral for aggressive maneuvers
-					if ((i < 2 && fabsf(gyro_filtered[i]) > 150.0f) ||
-						(i == 0 && fabsf(raw_input) > 0.2f)) {
+					if ((i < 2 && fabsf(gyro_filtered[i]) > settings.AcroZeroIntegralGyro) ||
+						(i == 0 && fabsf(raw_input) > settings.AcroZeroIntegralStick / 100)) {
```

Comment:
Agreed, but I was trying to follow the format of line 517; should I change 517 too?